custom_components/waste_collection_schedule/waste_collection_schedule/source/erlangen_hoechstadt_de.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import datetime
+from waste_collection_schedule import Collection # type: ignore[attr-defined]
+from waste_collection_schedule.service.ICS import ICS
+
+import urllib
+
+TITLE = "Landkreis Erlangen-Höchstadt"
+DESCRIPTION = "Source for Landkreis Erlangen-Höchstadt"
+URL = "https://www.erlangen-hoechstadt.de/"
+TEST_CASES = {
+    "Höchstadt": {"city": "Höchstadt", "street": "Böhmerwaldstraße"},
+    "Brand": {"city": "Eckental", "street": "Eckenhaid, Amselweg"},
+    "Ortsteile": {"city":"Wachenroth", "street": "Wachenroth Ort ink. aller Ortsteile"}
+}
+
+
+class Source:
+    def __init__(self, city, street):
+        self._city = city
+        self._street = street
+        self._ics = ICS()
+
+    def fetch(self):
+        city = urllib.parse.quote(self._city.upper())
+        street = urllib.parse.quote(self._street)
+        today = datetime.date.today()
+        year = today.year
+        r = requests.get(
```

Comment:
Please pass arguments in URLs via the `params` argument: https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls

I assume you can then also remove the urllib.parse.quote calls.

Reply:
Done, removing `urllib.parse.quote` worked as well.